inc/class-upgrade.php

```diff
@@ -730,6 +731,19 @@ private function upgrade_149() {
 		delete_option( 'wpseo_license_server_version' );
 	}
 
+	/**
+	 * Performs the 15.1 upgrade routine.
+	 *
+	 * @return void
+	 */
+	private function upgrade_151() {
+		$home_url = WPSEO_Options::get( 'home_url' );
+
+		if ( empty( $home_url ) ) {
+			WPSEO_Options::set( 'home_url', get_home_url() );
+		}
```

Comment:
Quick question: this is a new value we're creating in the `wpseo` option in this upgrade routine. Not sure I understand why we should check if it's empty? It's new, so it doesn't exist when when we try to get and check if empty.

Reply:
With all the issues I've ran in to with up- and downgrading versions, this seemed like the most sensible approach. If for some reason someone has the `home_url` set, downgrades and then upgrades again, any changes that were made, could be reset again. If anything, this is just to make sure we don't override anything pre-existing.